src/js/components/HealthBar.js

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import HealthBarMap from '../constants/HealthBar';
+import StatusBar from './StatusBar';
+
+class HealthBar extends React.Component {
+  renderToolTip() {
+    let {tasks, instancesCount} = this.props;
+
+    tasks = Object.keys(tasks).filter(function (task) {
```

Comment:
can we break this logic out into a method and give it a good name for what it's doing?